eth_utils/logging.py

```diff
@@ -57,13 +60,18 @@ def __new__(mcls, name, bases, namespace):
         return super().__new__(mcls, name, bases, assoc(namespace, "logger", logger))
 
     @classmethod
-    def _replace_logger_class(mcls, value: Type[logging.Logger]) -> "HasLoggerMeta":
+    def replace_logger_class(
+        mcls: Type[THasLoggerMeta], value: Type[logging.Logger]
+    ) -> Type[THasLoggerMeta]:
         # mypy can't tell this is a subclass of `HasLoggerMeta`  # noqa: E501
-        return type(mcls.__name__, (mcls,), {"logger_class": value})  # type: ignore
+        return type(mcls.__name__, (mcls,), {"logger_class": value})
 
-    def __getitem__(cls, value: Type[logging.Logger]) -> "HasLogger":
-        mcls = cls._replace_logger_class(value)
-        return mcls(cls.__name__, (cls,), cls.__dict__.copy())
+    @classmethod
+    def meta_compat(
+        mcls: Type[THasLoggerMeta], other: Type[type]
+    ) -> Type[THasLoggerMeta]:
+        # mypy doesn't recognize the bases for this class as valid
```

Comment:
I don't understand this comment. I was expecting to find a `type: ignore` on the next line. Is the comment not true anymore?